contracts/v0.8/MinerAPI.sol

```diff
@@ -23,11 +23,15 @@ contract MinerAPI {
         sectorSizesBytes[CommonTypes.SectorSize._64GiB] = 2 * (32 << 30);
     }
 
+    /// (Mock method) Sets the owner of a Miner, which will be returned via get_owner().
     function mock_set_owner(string memory addr) public {
         require(bytes(owner).length == 0);
         owner = addr;
     }
 
+    /// Returns the owner address of a Miner.
```

Comment:
Is this correct? should we use @return as it says here? 
https://docs.soliditylang.org/en/develop/natspec-format.html

Reply:
This is method docs, this is what the method does. But I agree we should document return types via `@return` tags too! Can we submit this patch and maybe you can submit a follow-up patch with the `@return` texts?

Reply:
Done!